translations/da.po

```diff
@@ -51482,7 +51482,7 @@ msgid "\"The Additional Information service is used to retrieve cover images. \"
 msgstr ""
 
 #: /admin/config/ting/covers/addi?render=overlay
-msgid "\"URL to the Additional Information webservice, e.g. \"http://moreinfo.addi.dk/2.10"
+msgid "\"URL to the Additional Information webservice, e.g. \"http://moreinfo.addi.dk/2.11"
```

Comment:
Normally translations is handled by DDB and this will be overwritten. There are on going work to setup translation servers for DDB CMS. So for now please revert this or rebase so it can be merged.

Reply:
Fixed